src/picongpu/include/simulation_defines/unitless/ionizerConfig.unitless

```diff
@@ -20,5 +20,6 @@
 
 #pragma once
 
+#include "particles/ionization/byField/ADK/AlgorithmADK.hpp"
```

Comment:
I would like to do all the renaming consistently in a later pull request if that is okay with you.